vagrant/provisioning/roles/activemq/tasks/main.yml

```diff
@@ -342,5 +391,9 @@
     name: activemq
     state: started
 
-    
+- name: Set Java alternatives to OpenJDK 8
```

Comment:
Martin, this activemq role was just for Dave's reference since he was doing an activemq upgrade in EEOC. This commit was just for reference and not a merge request.